zipkin-storage/cassandra3/src/test/java/zipkin/storage/cassandra3/CassandraUtilTest.java

```diff
@@ -95,6 +95,9 @@ public void annotationKeys_skipsBinaryAnnotationsLongerThan256chars() throws Exc
     )).build();
 
     assertThat(CassandraUtil.annotationKeys(span))
-        .containsOnly("web:aws.arn", "web:aws.arn:" + arn);
```

Comment:
ps I only now noticed that for a year we were indexing binaryAnnotation.key in the same space as annotation.value. This goes against all of our docs, and probably people just didn't notice.

Took it out in https://github.com/openzipkin/zipkin/pull/1292